src/Processors/QtiV2/In/DistractorRationaleResponseMapper.php

```diff
@@ -39,7 +37,7 @@ public function parseWithDistractorRationaleResponseComponent(RubricBlock $rubri
     
     private function getDomForXml($xml)
```

Comment:
It would make sense to me to also refactor this function so that we're using the `DOMHelper` class here too.

Reply:
Done